<launch>
    <!-- 输入图像缺省值改为 ZED 相机左右话题 -->>
    <arg name = "image_input", default = "/cv_camera/image_raw">
    
    <!-- 输入点云缺省值要改成 Robosense 雷达主题名 -->>
    <arg name = "cloud_input" default = "/velodyne_points"/>

    <!-- 图像点云融合输出话题名 -->
    <arg name = "fusion_topic" default = "/fusion_cloud">

    <!-- 图像点云融合节点 -->
    <node name = "lidar_camera_fusion_node" pkg = "lidar_camera_fusion" type = "lidar_camera_fusion" output = "screen">
        <param name = "image_input" type = "str" value = "$(arg image_input)" />
        <parma name = "cloud_input" type = "str" value = "$(arg cloud_input)" />
        
        <param name = "fusion_topic" type = "str" value = "$(arg fusion_topic)" />
    </node>

</launch>